docs/2-Collection_Containers.adoc

```diff
@@ -965,6 +965,90 @@ They are named *{IntHashSet}*, *{FloatHashSet}* etc.
 *{BooleanHashSet}* is implemented using a single integer to hold one of four possible states: ([], [F], [T], or [T, F]).
 All other sets use open addressing and quadratic probing.
 
+.There are different ways you can create primitive sets, which are listed as below.
+- You can create primitive sets using **newSetWith** method.
+- You can initialize primitive sets using constructor argument. There are 5 different overloaded constructor are available.
```

Comment:
"There are 5 different overloaded constructor are available." -> "There are several overloaded constructors available."

Reply:
Thank you for suggestions, I have updated adoc with suggested changes.